video_xblock/static/css/brightcove.css

```diff
@@ -643,3 +643,11 @@ div.vjs-control-bar > div.vjs-menu-button.vjs-menu-button-popup.vjs-control.vjs-
 div.vjs-control-bar > div.vjs-custom-caption-button.vjs-menu-button.vjs-menu-button-popup.vjs-control.vjs-button.icon.fa.fa-cc.vjs-singleton.vjs-control-enabled .vjs-menu {
     display: none !important;
  }
+
+div.vjs-control-bar > div.vjs-menu-button.vjs-menu-button-popup.vjs-control.vjs-button.icon.fa.fa-cc.vjs-singleton > div > ul,
+body .video-js .fa-quote-left .vjs-menu,
+body .video-js .fa-cc .vjs-menu,
+body .vjs-v6 .vjs-custom-caption-button .vjs-menu-content,
+body .vjs-v6 .vjs-custom-transcript-button .vjs-menu-content {
+    display: none !important;.
```

Comment:
@wowkalucky - there is an extra `.` here.  Should be removed

Reply:
@wowkalucky Thanks!  I merged, but it looks like your approach to deal with the duplicate menus was to just never show them when hovering over the caption and transcript buttons.  Did you try to deal with the issue while still maintaining the caption and transcript hover behavior?  We are going to check with our sponsoring client whether this is OK, but it is less intuitive for the user if the languages don't appear on the caption and transcript button.  What kind of additional work do you think it would be to maintain the original behavior and also avoid duplicate menus?